### PR TITLE
added split test for bigquery

### PR DESCRIPTION
**Pull request by stevenhurwitt**
## What is this? 
Added a split test for bigquery

## What changes in this PR? 
Checked bigquery tests with `docker-compose exec testxdb test` and generated new docs with `docker-compose exec testxdb docs` (also ran coverage).

## How does this impact our users?
Allows split macro to be used in bigquery.

## PR Quality
- [x] does `docker-compose exec testxdb test` run successfully?
- [x] does `docker-compose exec testxdb docs` build docs without errors?
- [x] does `docker-compose exec testxdb coverage` pass coverage standards?
- [x] did you leave the codebase better than you found it? 

@Health-Union/hu-data make sure to include a version tag in the merge commit!

### CODE REVIEW

File: docs/macros.md [hunk 1]
```diff
@@ -173,6 +173,18 @@ applies the weird escape sequences required for bigquery and snowflake
 **Returns**: 
 ##### Supports: _Postgres, Snowflake, BigQuery, Redshift_
 ----
+### [array_contains](../macros/array_contains.sql)
```
Comment (eculicny) [inline docs/macros.md:176]: Theoretically, this should have been updated after the PRs for it were merged. It looks like the `commit docs` step is actually failing. Added issue for it: #67 

Thanks for including the updated docs!

File: test_xdb/models/under_test/split_test.sql [hunk 2]
```diff
@@ -1,12 +1,14 @@
 /* TODO split is supported for bigquery. tests should support that */
```
Comment (eculicny) [inline test_xdb/models/under_test/split_test.sql:1]: Since this fixes it, might as well remove the `TODO` comment along with these changes.

**Review by eculicny — CHANGES_REQUESTED** [review]
Looks good!

File: test_xdb/models/under_test/split_test.sql (hunk 2 above)
Comment (stevenhurwitt) [reply test_xdb/models/under_test/split_test.sql]: whoops, just pushed up a commit to remove it!

File: docs/macros.md (hunk 1 above)
Comment (stevenhurwitt) [reply docs/macros.md:176]: Ok, good to know. I was unsure whether to include it or not...

**Review by jedibrillo — APPROVED** [review]

**Review by eculicny — APPROVED** [review]

**Pull request merged**